src/test/java/com/google/api/codegen/packagegen/testdata/library.proto

```diff
@@ -230,24 +297,34 @@ message ListBooksResponse {
 // Request message for LibraryService.UpdateBook.
 message UpdateBookRequest {
   // The name of the book to update.
-  string name = 1;
+  string name = 1 [
```

Comment:
Oh, I see, it is because this is wrong (which is unrelated to this PR).

Meh, okay.